pkg/apis/eksctl.io/v1alpha5/assets/schema.json

```diff
@@ -466,6 +466,12 @@
           "x-intellij-html-description": "arbitrary metadata ignored by <code>eksctl</code>.",
           "default": "{}"
         },
+        "disableStackPrefix": {
```

Comment:
So, this is only for stacks, right? Not other resources `eksctl` prefixes, such as roles, service accounts, etc...?

That makes sense now. I see why this pr is as small as this. :D

Reply:
Yeah, I was under the impression that things like roles and service accounts could have their names overridden already too. Since the idea was more around allowing users to use their own naming conventions for cloudformation stacks specifically I honestly didn't really dig around much beyond that.

Reply:
Well, yes and no. You can't override some of the roles and you can't override many of the internal resources like, subnet routing or whatever, which gets created. Mostly roles...

Reply:
Ah cool, I can dig around to see what I can find that could benefit from inheriting the disabled prefix :+1: